### PR TITLE
get rid of the name Kubermatic for resources visible to the end user

### LINKED ISSUE #2533: Replace word `kubermatic` with `kubernetes` in all cloud provider resources
**User Story**
If a customer want's to use Kubermatic & "whitelabel" it, still all cloud provider resources for user clusters have the prefix `kubermatic`.

This prefix must be replaced with the word `kubernetes`.

**Acceptance criteria**
- [ ] `kubermatic` does not get used for any cloud provider resource anymore
- [ ] This change is backwards compatible

### PR DESCRIPTION
**Pull request by mrIncompetent**
**What this PR does / why we need it**:
Removes the name "kubermatic" from all resources visible by end users.

Fixes #2533

**Release note**:
```release-note
Avoid the name "kubermatic" for resources visible by end users
```

### CODE REVIEW

**Comment by mrIncompetent** [comment]
/hold
i need to update the image

File: addons/openvpn/vpn-client-serviceaccount.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v1
```
Comment (alvaroaleman) [inline addons/openvpn/vpn-client-serviceaccount.yaml:1]: ~~Maybe do the same for the openshift addon?~~

**Comment by alvaroaleman** [comment]
/lgtm
/approve
/hold

to fix up the addon image tags

**Comment by mrIncompetent** [comment]
@alvaroaleman PTAL

I've updated the addon image tags + updated the fixtures

**Pull request merged**